apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -605,7 +593,6 @@ module.exports = connect(function propsFromStore(state) {
     isEmbedView: state.pageConstants.isEmbedView,
     embedViewLeftOffset: state.pageConstants.nonResponsiveVisualizationColumnWidth + VIZ_TO_INSTRUCTIONS_MARGIN,
     isMinecraft: !!state.pageConstants.isMinecraft,
-    hasContainedLevels: state.pageConstants.hasContainedLevels,
```

Comment:
Another example of Chris trying to make contained levels more flexible. We don't currently have any of these in CSF or immediate plans to add them. I'd rather not have unused/untested code. If we need these in CSF in the future, it shouldn't be too difficult to add them.